Brain/Brain.ino

```diff
@@ -99,5 +104,16 @@ void loop() {
 bool write_to_SD(String message){
   // every reading that we get from sensors should be written to sd and saved.
   // TODO: Someone's code here
-  return false;
+    if (!SD.begin(BUILTIN_SDCARD))
+        return false;
+   
+    File myFile = SD.open(file_name.c_str(), FILE_WRITE);
+
+    if (myFile) {
+      myFile.println(message);
+      myFile.close();
```

Comment:
Instead of opening the file each time, can we setup some sort of buffer that will wait until you have like 10 messages or something, and then write them all together. You can look up how to do this.